### PR TITLE
Log email voters

### PR DESCRIPTION
**Pull request by edmarmartineli**
I had a problem with an election administrator to know if he had called the voters to vote, so I decided to make a log about it.

### CODE REVIEW

File: helios/views.py [hunk 1]
```diff
@@ -1315,6 +1315,12 @@ def voters_email(request, election):
   if not template in [t[0] for t in TEMPLATES]:
     raise Exception("bad template")
 
+  template_tyte = ('','')
```
Comment (benadida) [inline helios/views.py]: maybe we just need to change the templates to be a hashmap instead of this code?

File: helios/views.py [hunk 2]
```diff
@@ -1379,6 +1388,17 @@ def voters_email(request, election):
 
         tasks.voters_email.delay(election_id = election.id, subject_template = subject_template, body_template = body_template, extra_vars = extra_vars, voter_constraints_include = voter_constraints_include, voter_constraints_exclude = voter_constraints_exclude)
 
+        # get send_to
+        send_to_choices = email_form.fields['send_to'].choices
```
Comment (benadida) [inline helios/views.py]: why not just take the form value?

Comment (edmarmartineli) [reply helios/views.py]: I need to get the description of the value. If I get the form value, the text recorded in the log is not very friendly.

File: helios/views.py (hunk 1 above)
Comment (edmarmartineli) [reply helios/views.py]: I'm not sure how to do this, but it's a better way. I will try.

**Pull request closed without merging**